### ISSUE TITLE
scenario nep2035 only wind and pv capacities

### ISSUE THREAD

**Issue by thofritz** [openego/eDisGo#158]
Using eDisGo for simulating some MVGDs, the tool just imports wind and pv from the oedb, even though the scenario is set to "nep2035". In the scenario, get_gen_info(edisgo.network, fluctuating=False) returns, that there are only wind and pv capacities. What happened with the capcaities of the other energy sources like biomass and run_of_river?

**Comment by birgits** [comment]
@thofritz you asked a very good question. Other energy sources were neglected in the import, meaning the Status Quo capacities are used. 
The main reason for that is that IDs of power plants were changing between the different scenarios which lead to difficulties in updating the power plants in the distribution grids (in the high and extra high voltage levels capacities this did not pose a problem wherefore the capacities are updated correctly). In issue #59 we evaluated that the share of conventional generators connected to the medium and low voltage is quite low and their impact on grid expansion needs negligible compared to renewable capacities (see [this comment](https://github.com/openego/eDisGo/issues/59#issuecomment-385376958)). 
Run-of-river capacities are not significantly changing as well (see [figure 1 in this paper](https://www.mdpi.com/1996-1073/12/11/2091/htm) that gives the total installed capacities for all scenarios and voltage levels). 
Concerning the biomass capacities I'm afraid I don't have a satisfying answer, as we did not evaluate the impact. From Status Quo to NEP 2035 capacities didn't increase significantly as the figure shows. However, in the eGo 100 scenario capacities doubled which could be significant for grid expansion costs in the medium and low voltage.
